frontend/src/elm/Main/View.elm

```diff
@@ -0,0 +1,228 @@
+module Main.View exposing (view)
+
+import Html exposing (Html, div, p, text, button, input, form, select, option, label)
+import Html.Attributes exposing (id, class, name, placeholder, action, type_, checked, value, for, style)
+import Html.Events exposing (onClick, onInput, onCheck, onSubmit)
+import Array
+import Main.Msgs exposing(Msg(..))
+import Main.Models exposing (..)
+
+
+--for filter subjects
+isMajor : Subject -> Bool
```

Comment:
Commenting right above `isMajor` may seem like the comment is only for `isMajor`.
To indicate commenting as section, it would be better to add newline like
```elm
--for filter subjects

isMajor : Subject -> Bool
```